src/scores/stats/confidence_intervals_impl.py

```diff
@@ -0,0 +1,383 @@
+"""
+Functions for calculating confidence statistics
```

Comment:
If these methods are entirely related to Diebold Mariano (even if only for the moment), I would suggest renaming this file to diebols_mariano_impl.py

Reply:
Renamed